api/src/main/java/org/apache/iceberg/transforms/PartitionSpecVisitor.java

```diff
@@ -86,6 +86,10 @@ default T alwaysNull(int fieldId, String sourceName, int sourceId) {
     throw new UnsupportedOperationException("Void transform is not supported");
   }
 
+  default T alwaysNull(String sourceName, int sourceId) {
+    throw new UnsupportedOperationException("Void transform is not supported");
```

Comment:
Nit: Copy paste error? This is not the Void transform. Should be `Always null transform is not supoprted` (if needed at all).

Reply:
Nope this was meant to be intentional. Each method here has a pair that doesn't take a `int fieldId`, and this was missing for the `alwaysNull` method.
I believe this is actually the void transform, which other method should it be if not this one?